scenarios3/10_Gehenna.cfg

```diff
@@ -275,16 +279,22 @@
             y=2
             side=1
         [/filter]
+#ifndef HARD
         [sound]
             name=gold.ogg
         [/sound]
+#endif
         [message]
             speaker=unit
+#ifdef HARD
+            message= _"It's empty!"
```

Comment:
It would be hilarious to add a remark that someone had embezzled their entire treasury and that it's a shame that you can't just wait for them to go bankrupt.

Reply:
try that